gapic/src/main/com/google/gapid/perfetto/models/CpuTrack.java

```diff
@@ -37,6 +37,7 @@
 import com.google.gapid.perfetto.views.State;
 
 import java.util.Arrays;
+import org.eclipse.swt.SWT;
```

Comment:
Unused?

Reply:
Thanks for noting, have removed unused imports in new commits.